README.md

```diff
@@ -60,17 +60,42 @@ listening for API requests and provisioning volumes accordingly.
 
 ### Architecture and Operation
 
-The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct
+The PMEM-CSI driver can operate in two different device modes: *LVM* and *direct*. This table contains an overview and comparison of those modes. There is more detailed modes explanation down in following paragraphs.
 
-### DeviceMode:LVM
+|                   |LVM                    |direct              |
```

Comment:
Nitpick comment: some paragraphs in table start with uppercase and others don't, can we homogenize this?